apps/src/templates/progress/FocusAreaIndicator.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * A component that adds a sort of ribbon looking focus indicator to the side
+ * of your row. Note: This is English only.
```

Comment:
Ah, answers my question from before.

Also, no need to sound hesitant here. "A focus indicator ribbon for the side of your row."